net/graphsync_fetcher.go

```diff
@@ -0,0 +1,180 @@
+package net
+
+import (
```

Comment:
nit: We use the following import ordering (per [conventions and style](https://github.com/filecoin-project/go-filecoin/blob/master/CONTRIBUTING.md#conventions-and-style)):
```
import (
        [stdlib packages, alpha-sorted]
        <single, empty line>
        [external packages]
        <single, empty line>
        [go-filecoin packages]
)
```